reducers/messageReducer.js

```diff
@@ -13,16 +13,18 @@ function messageReducer(state = messageInitialState, action) {
                 ...state,
                 messages: payload
             };
-        case types.CREATE_MESSAGE:
+        case types.CREATE_MESSAGE: {
+            const newMessage = { ...payload, reactions: [] };
             return {
```

Comment:
I think you should be able to remove `: []` in `reactions: []` given the suggested change on the PR https://github.com/emote-chat/server/pull/31.

Reply:
this is when you add a new message, so i think we'd have to modify what gets returned from the create endpoint first. since it's a new message, there wouldn't be any reactions yet, so that's why i set it to an empty array.

Reply:
i updated the api when creating a new message to just return the empty array instead!